### PR TITLE
Prepare v4

### PR DESCRIPTION
**Pull request by Antynea**
* Readme updated
* localization no longer supported
* New systemd service.
* 41_snapshots-btrfs, ahead updated
* Makefile added

### CODE REVIEW

**Comment by Antynea** [comment]
@maximbaz 
Sorry, too many update for readme file....

File: 41_snapshots-btrfs [hunk 1]
```diff
@@ -6,30 +6,36 @@
...
 #
-# How to customize it:
-# - Refer to config for the list of available options and their default values.
-# - Place your configurations to /etc/default/grub-btrfs/config.
+# Installation:
+# - Copy 41_snapshots-btrfs to /etc/grub.d/. (make sure the permissions are 755)
+# - Copy config to /etc/default/grub-btrfs/. (make sure the permissions are 644)
+# - Copy grub-btrfs.path to /etc/systemd/system/ or /lib/systemd/system/. (make sure the permissions are 644)
+# - Copy grub-btrfs.service to /etc/systemd/system/ or /lib/systemd/system/. (make sure the permissions are 644)
+# - Generate your Grub menu after installation for the changes to take effect. (on Arch linux use grub-mkconfig -o /boot/grub/grub.cfg)
```
Comment (maximbaz) [inline 41_snapshots-btrfs]: Are you sure you want to describe it in text? If Makefile changes, need to remember to update this... Maybe just say:

```
Run `make install` or look into Makefile for instructions on where to put each file.
```

File: 41_snapshots-btrfs [hunk 2]
```diff
@@ -6,30 +6,36 @@
...
-# Automatically update grub
-#  If you would like grub to automatically update when Snapper timeline snapshots and cleanups occur, simply install 10-update_grub.conf in the following locations:
-# - /etc/systemd/system/snapper-timeline.service.d/
-# - /etc/systemd/system/snapper-cleanup.service.d/
-# Once the configuration files are in place, systemctl daemon-reload should be run to reload the units and make the changes active.
+# Automatically update Grub
+#  If you would like Grub to automatically update when a snapshots is made or deleted:
+# - Mount your subvolume which contains snapshots to /.snapshots
+# - Use systemctl start/enable grub-btrfs.path
+#  grub-btrfs.path automatically (re)generate grub.cfg when a modification appear in /.snapshots
```
Comment (maximbaz) [inline 41_snapshots-btrfs]: grub-btrfs.path _will_ automatically

File: 41_snapshots-btrfs [hunk 3]
```diff
@@ -47,6 +53,7 @@ grub_btrfs_config="${sysconfdir}/default/grub-btrfs/config"
 
 [[ -f "$grub_btrfs_config" ]] && . "$grub_btrfs_config"
 . "$datarootdir/grub/grub-mkconfig_lib"
+. "${sysconfdir}/default/grub"
```
Comment (maximbaz) [inline 41_snapshots-btrfs]: are you sure this line is needed? I thought we _only_ configure via our own config

File: README.md [hunk 4]
```diff
@@ -82,44 +88,28 @@ Add this lines to /etc/default/grub:
 
 	(Snapper's config name to use)
 
-* GRUB_BTRFS_DISABLE="true"
+* GRUB_BTRFS_DISABLE="false"
 
-	(Disable grub-btrfs)
+	(Disable Grub-btrfs)
```
Comment (maximbaz) [inline README.md]: keep lowercase, I guess

File: 41_snapshots-btrfs (hunk 1 above)
Comment (Antynea) [reply 41_snapshots-btrfs]: you are right :)

File: 41_snapshots-btrfs (hunk 2 above)
Comment (Antynea) [reply 41_snapshots-btrfs]: my bad ...

File: 41_snapshots-btrfs (hunk 1 above)
Comment (maximbaz) [reply 41_snapshots-btrfs]: Don't forget to do the same in README 😉 Otherwise looks very good!

**Comment by Antynea** [comment]
@maximbaz 
All conversation are resolved, i think.
Are you ok for merging ?

**Comment by maximbaz** [comment]
Looks good to me, merge 😉

**Comment by Antynea** [comment]
@maximbaz 
Done !
New release available :)

**Pull request merged**